templates/init-registry-org.py.j2

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+from argparse import ArgumentParser, RawTextHelpFormatter
+from os import environ
+import json
+import logging
+import requests
+
+HELP_DEFINITION='''
+Script to initiate Gitea Image registry organization
+'''
+HELP_EXAMPLE='''
+Example:
+\n\t./init-regisitry-org.py 
+  --org org1,org2,org3 \
+  --url https://example.com/gitea \
+  --user username --password password
+'''
+
+headers= {
+    'Content-type': 'application/json'
+}
+
+def create_org(url, org, basic):
+    org_data={'visibility': 'limited'}
+    org_data['username'] = org
+    logging.info('Creating organisation %s' % org)
+    logging.debug('Argument:%s' % json.dumps(org_data))
+    res=requests.post('%s/api/v1/orgs' % url, auth=basic,headers=headers,data=json.dumps(org_data))
+    res.raise_for_status()
+    return res
+
+def prepare_credentials(args):
+    username=None
+    password=None
+    if args.username is not None:
+        logging.info('Loading username from parameter')
+    if args.password is not None:
+        logging.warn('Loading password from parameter, not recommended')
+    if environ.get('GITEA_USERNAME') is not None:
+        logging.info('Loading username from environment variable')
+        username=environ.get('GITEA_USERNAME')
+    if environ.get('GITEA_USERNAME') is not None:
+        logging.info('Loading password from environment variable')
+        username=environ.get('GITEA_PASSWORD')
```

Comment:
You can avoid all of this complexity by just using the `default` argument:
```python
parser.add_argument('-u', '--username', default=environ.get('GITEA_USERNAME'),
                    help='Gitea API username')
parser.add_argument('-p', '--password', default=environ.get('GITEA_PASSWORD'),
                    help='Gitea API password')
```
And then just assert certain requirements for combined arguments at the end of `parse_args`:
```python
    args = parser.parse_args()
    assert args.username and args.password, parser.error('No API username or password!')
    return args
```